Source/WTF/wtf/PlatformHave.h

```diff
@@ -52,6 +52,17 @@
 #define HAVE_ARM_IDIV_INSTRUCTIONS 1
 #endif
 
+#if CPU(ADDRESS64)
```

Comment:
moved due to header inclusion order

Reply:
I guess technically this is a PLATFORM() now, not OS(), but I wasn't sure if I should change it.